src/test/kotlin/com/jetbrains/snakecharm/inspections/SmkSectionRedeclarationInspectionTest.kt

```diff
@@ -0,0 +1,21 @@
+package com.jetbrains.snakecharm.inspections
+
+import com.intellij.codeInspection.LocalInspectionTool
+import com.jetbrains.snakecharm.fixtures.SnakemakeInspectionTestCase
+
+class SmkSectionRedeclarationInspectionTest : SnakemakeInspectionTestCase() {
+    override val inspectionClass: Class<out LocalInspectionTool>
+        get() = SmkSectionRedeclarationInspection::class.java
```

Comment:
Here simpler is `override val inspectionClass = SmkSectionRedeclarationInspection::class.java`. But please re-write this using cucumber features.